frontend/src/components/modules/Login/Login.tsx

```diff
@@ -14,88 +22,174 @@ interface FormData {
   password: string;
 }
 
+interface ErrorMessage {
+  message: string;
+}
+
```

Comment:
I would move this to somewhere more common, since all of the request responses have the same body in case of an error.

Reply:
alright.

Reply:
Showing error has been moved to common component section.